src/plugins/GUITestBase/src/tests/common_scenarios/msa_editor/colors/GTTestsMSAEditorColors.cpp

```diff
@@ -45,12 +45,27 @@ namespace U2 {
 namespace GUITest_common_scenarios_msa_editor_colors {
 using namespace HI;
 
+void checkColor(HI::GUITestOpStatus& os, const QPoint& p, const QString& expectedColor, int Xmove = 0, int Ymove = 0) {
```

Comment:
We already have methods to check colors like this.
Your impl is not safe because may overlap with a letter inside the cell and the shape of letter depends on OS, font, local styles that are different for Win/Linux/Mac